lib/cli.ts

```diff
@@ -9,6 +9,17 @@ import ConsoleUI from '../types/console-ui';
 import WatchDetector from 'watch-detector';
 import UI from 'console-ui';
 
+enum EnvironmentType {
+  PRODUCTION = 'production',
+  DEVELOPMENT = 'development',
+}
+
+enum WatcherType {
```

Comment:
this seems good

Reply:
it's predefined sane watcher options 

![image](https://user-images.githubusercontent.com/1360552/134964024-9f64339c-70e7-48b0-b749-be2815a7075c.png)